openmetadata-ui/src/main/resources/ui/public/locales/en-US/Database/workflows/metadata.md

```diff
@@ -4,31 +4,59 @@ DatabaseService Metadata Pipeline Configuration.
+Schema filter patterns are used to control whether or not to include schemas as part of metadata ingestion.Enter the regex pattern form the for including or excluding the database schema. Checkout [this](https://docs.open-metadata.org/connectors/ingestion/workflows/metadata/filter-patterns/database#database-filter-pattern) document for examples on schema filter pattern
 $$
 
 $$section
-### Schema Filter Pattern $(id="schemaFilterPattern")
+### Table Filter Pattern $(id="tableFilterPattern")
 
-Regex to only fetch tables or databases that matches the pattern.
+Table filter patterns are used to control whether or not to include tables as part of metadata ingestion.Enter the regex pattern form the for including or excluding the tables. Checkout [this](https://docs.open-metadata.org/connectors/ingestion/workflows/metadata/filter-patterns/database#table-filter-pattern) document for examples on table filter pattern
 $$
 
 $$section
-### Table Filter Pattern $(id="tableFilterPattern")
+### Use FQN For Filtering $(id="useFqnForFiltering")
+
+This flag set when you want to apply the filter on fully qualified name (e.g service_name.db_name.schema_name.table_name) instead of applying the filter to raw name of entity (e.g table_name). This Flag is useful in scenario when you have schema with same name in different databases or table with same name in different schemas and you want to filter out one of them. This will be exaplined further in detail in this document. Checkout [this](https://docs.open-metadata.org/connectors/ingestion/workflows/metadata/filter-patterns/database#table-filter-pattern) document for examples on how to use this field.
 
- Regex exclude tables or databases that matches the pattern.
 $$
 
 $$section
-### Database Filter Pattern $(id="databaseFilterPattern")
+### Include Views $(id="includeViews")
+
+Set the Include views toggle to control whether or not to include views as part of metadata ingestion.
 
-Regex to only fetch databases that matches the pattern.
 $$
+
+$$section
+### Include Tags  $(id="includeTags")
+
+Set the Include tags toggle to control whether or not to include tags as part of metadata ingestion.
+
+$$
+
+$$section
+### Enable Debug Logs $(id="loggerLevel")
+
+Enabling debug logs tracks error messages during ingestion for troubleshooting.
+$$
+
+$$section
+### Mark Deleted Tables $(id="markDeletedTables")
+
+This is an optional configuration for enabling soft deletion of tables. When this option is enabled, only tables that have been deleted from the source will be soft deleted, and this will apply solely to the schema that is currently being ingested via the pipeline. Any related entities such as test suites or lineage information that were associated with those tables will also be deleted.
```

Comment:
I would give some examples here as well explaining one scenario
1. we ingest X
2. gets deleted, etc